jetty-http-spi/src/main/java/org/eclipse/jetty/http/spi/DelegatingThreadPool.java

```diff
@@ -63,16 +63,16 @@ public boolean tryExecute(Runnable task)
     }
 
     @Override
-    public boolean isUseVirtualThreads()
+    public Executor getVirtualThreadsExecutor()
     {
-        return VirtualThreads.isUseVirtualThreads(_executor);
+        return VirtualThreads.getVirtualThreadsExecutor(_executor);
     }
 
     @Override
-    public void setUseVirtualThreads(boolean useVirtualThreads)
```

Comment:
Do we have a release with this name?
If so, this can't be removed, only deprecated.

Reply:
It's not removed, it's `default` in the interface so it's inherited anyway.